src/main/java/raisetech/StudentManagement/data/Student.java

```diff
@@ -34,4 +34,8 @@ public class Student {
   private String gender;
   private String remark;
   private boolean isDeleted;
+
+  public Student(int id) {
```

Comment:
IDのみをもったStudentのデータというのは不自然なので、全フィールドを引数にとるようなコンストラクタを使うのがいいかと。
```suggestion
  public Student(int id, String fullName, String furigana, String nickname, String emailAddress, String area, Integer age (省略)) {
```

Reply:
chap41の課題をやる際に@AllArgsConstructorを追加したのですが、こちらでも問題ないでしょうか？
4e50483f7c9fa5345ad668202eb274a4dd878b1c